exchangelib/extended_properties.py

```diff
@@ -264,3 +264,10 @@ class ExternId(ExtendedProperty):
     property_type = 'String'
 
     __slots__ = ExtendedProperty.__slots__
+
+
+############# Nylas Extended Porperties ##############
+class Flag(ExtendedProperty):
```

Comment:
what do you think about adding a comment above this saying:

```
# This is the definition for the 'completed' and 'followup' flag 
# you can add to items in Outlook:
# https://pypi.org/project/exchangelib/
```

Reply:
I could do that, I'll track down the right doc for it